### PR TITLE
Instantiate from mama-ng-control subscriptions

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: contentstore/urls.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+router.register(r'schedule', views.ScheduleViewSet)
+router.register(r'messageset', views.MessageSetViewSet)
+router.register(r'message', views.MessageViewSet)
+router.register(r'binarycontent', views.BinaryContentViewSet)
+
+# Wire up our API using automatic URL routing.
+# Additionally, we include login URLs for the browseable API.
+urlpatterns = [
+    url(r'^api/v1/', include(router.urls)),
+    url(r'^message/(?P<pk>.+)/content$',
```
Comment (imsickofmaps) [inline contentstore/urls.py]: these two also need pre-pending with version

File: subscriptions/models.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    messageset_id = models.IntegerField(null=False, blank=False)
+    next_sequence_number = models.IntegerField(default=1, null=False,
+                                               blank=False)
+    lang = models.CharField(max_length=6, null=False, blank=False)
+    active = models.BooleanField(default=True)
+    completed = models.BooleanField(default=False)
+    schedule = models.IntegerField(default=1)
+    process_status = models.IntegerField(default=0, null=False, blank=False)
+    metadata = JSONField(null=True, blank=True)
+    created_at = models.DateTimeField(auto_now_add=True)
```
Comment (imsickofmaps) [inline subscriptions/models.py]: add the `created_by` and `updated_by` please

**Comment by imsickofmaps** [comment]
Minor things, plus pep8 then :+1:

**Pull request merged**